usrsctplib/usrsctp.h

```diff
@@ -435,6 +435,8 @@ struct sctp_send_failed_event {
 /* flag that indicates state of data */
 #define SCTP_DATA_UNSENT  0x0001	/* inqueue never on wire */
 #define SCTP_DATA_SENT    0x0002	/* on wire at failure */
+#define SCTP_DATA_LAST_FRAG 0x01 /* tail part of the message could not me sent */
+#define SCTP_DATA_NOT_FRAG 0x03 /* complete message could not be sent */
```

Comment:
I would prefer to have these definitions after
```
/* Flags that go into the sinfo->sinfo_flags field */
```
and before
```
#define SCTP_NOTIFICATION     0x0010 /* next message is a notification */
```
because both are used in `sinfo_flags`.

Reply:
Sure thing.